solr/CHANGES.txt

```diff
@@ -136,6 +136,8 @@ Bug Fixes
 * SOLR-17093: Collection restore API command now returns "requestid" when executed asynchronously like other APIs
   (Tomás Fernández Löbbe)
 
+* SOLR-16952: In distributed i.e. multi-shard cloud mode returned dense vector 'fl' fields are now a list of numbers instead of strings. (Qais Qadri, Christine Poerschke)
```

Comment:
My understanding is that this does not impact the search -- in fact one can do a non-vector search and request the vector field in the `fl` -- the search happens on the shards and the merging is based on scores. The type in the response being non-numeric would impact on clients using that part of the response _and_ from re-reading the https://lists.apache.org/thread/xnfrxvm0p8yknrrszcqn0dd0vfg3wtvj user mailing list thread is appears that it _might_ also impact debug/explain functionality, which from vague memory would run on the federating node rather than on the shard nodes.

Also tagging @alessandrobenedetti here for awareness and I'll also reply on the mailing list thread.